pygenometracks/tracksClass.py

```diff
@@ -369,6 +369,10 @@ def check_file_exists(track_dict, tracks_path):
                 file_field_name = key
         # for file_field_name in ['boundaries_file', 'file', 'categories_file']:
         #     if file_field_name in track_dict:
+                if file_field_name == 'boundaries_file':
```

Comment:
Can you please add a comment for us, so we remove that line of code if we go towards 1.0.

Reply:
Done